.changeset/solid-squids-hunt.md

```diff
@@ -0,0 +1,14 @@
+---
+"@fluidframework/container-definitions": minor
+"@fluidframework/container-loader": minor
+---
+---
+"section": deprecation
+---
+
+Deprecate the Loader class and provide alternatives. Also deprecated the IHostLoader interface.
+
+Deprecate the Loader class and IHostLoader interface and instead provide standalone apis so that Host does not need to
+take responsibility of creating the Loader object. Rather host can directly use apis like {@link @fluidframework/container-loader#resolve}
```

Comment:
i'm not sure we can use js comment style links in our change sets and get good behavior. you also renamed resolve to resolveContainer